test/functions/whereClauses/whereClauseParamArgMismatch3.good

```diff
@@ -1,2 +1,2 @@
 whereClauseParamArgMismatch3.chpl:10: error: unresolved call 'foo(1.2)'
-whereClauseParamArgMismatch3.chpl:1: note: candidates are: foo(param x: uint(8))
+whereClauseParamArgMismatch3.chpl:1: note: candidates are: foo(param x: uint(64))
```

Comment:
But anyways, I'm happy to let you clean stuff after the fact if you want, but I have no problem leaving the tests as they are.